docs/version-history/feature-versions-23-x.rst

```diff
@@ -17,6 +17,11 @@ Release Notes:
 
 New Features:
 
+- On Linux systems with cgroup v1 enabled, HTCondor now uses the device cgroup
+  to hide GPUs that are not provisioned by the system.  This can be disabled
```

Comment:
```suggestion
  to hide GPUs that it did not provision.  This can be disabled
```

Reply:
We call them "AssignedGPUs" in the slot ad, and "the system" is either wrong or ambiguous.